docs/Lifecycle/ChangeControlBoard.md

```diff
@@ -0,0 +1,71 @@
+# **Change Control Board (CCB).**
+# Roles and responsibilities.
+
+CCB is a committee to which following responsibilities are delegated by stakeholders of SDL:
+1. Review and approve proposals made in scope of SDL Evolution
+2. Confirm delivery for readiness
+
+## **Roles**
+
+CCB include following roles:
+- Business Analyst (BA)
+- Architect
+- Test Lead Engineer
+- Developer Lead Engineer
+- CCB Group Manager
+
+### Business Analyst
+
+Following are responsibilities of BA:
+1. Proposal review
+	* Provide approval on proposal specification readiness
+	* Create set of requirements issues in respective repositories
+	* Create set of technical tasks in respective repositories
+	* Maintain links between proposals, requirements and technical tasks
+
+2. Delivery acceptance (overall review and acceptance)
+
+### Architect
+
+Following are responsibilities of Architect:
+1. Proposal review
+	* Review and approve architecture impact
+
+2. Delivery acceptance
+	* Review and approve SAD updates
+	* Review and approve SDD updates
+
+### Test Lead Engineer
+
+Following are responsibilities of Test Lead Engineer:
+1. Proposal review
+	* Review and approve acceptance criteria
+
+2. Delivery acceptance
+	* Review and approve integration scripts
+
+### Developer Lead Engineer
+
+Following are responsibilities of Developer Lead Engineer:
+1. Proposal review
+	* Review and approve applicability/relevance of the proposal
+
+2. Delivery acceptance
+	* Review and approve UTs and major code changes
```

Comment:
I suggest to add something like `Review code for following Design and Architectural solutions`
The main idea, that code shall not break or conflict with Arch/Design approaches.